File: help/eng_gb/exercicios_da_ficha_2/Ficha02_015/Ficha02_015.cs
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha02_015 {

    //Main Function
    static void Main() {
        Console.Write("1st numer :" );
        int n1 = int.Parse(Console.ReadLine());
        Console.Write("2nd number :" );
        int n2 = int.Parse(Console.ReadLine());
        Console.Write(n1 + " + " + n2 + " = " + ( n1 + n2 ) );
        Console.Write("\n" + n1 + " - " + n2 + " = " + ( n1 - n2 ) );
        Console.Write("\n" + n1 + " * " + n2 + " = " + ( n1 * n2 ) );
        Console.Write("\n" + n1 + " / " + n2 + " = " + ( n1 / n2 ) );
        Console.Write("\n" + n1 + " % " + n2 + " = " + ( n1 % n2 ) );
        Console.Write("\n" + n1 + " ^ " + n2 + " = " + ( Convert.ToInt64( Math.Pow( n1 , n2 ) ) ) );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 set 21 22:52

File: help/eng_gb/exercicios_da_ficha_2/Ficha02_015/Ficha02_015.js
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("1st numer :" );// print text
    n1 = parseInt(prompt("1st numer :" ,"0") );
    
    document.write(n1 + "\n");// print input 
    document.write("2nd number :" );// print text
    n2 = parseInt(prompt("2nd number :" ,"0") );
    
    document.write(n2 + "\n");// print input 
    document.write(n1 + " + " + n2 + " = " + ( n1 + n2 ) );
    document.write("\n" + n1 + " - " + n2 + " = " + ( n1 - n2 ) );
    document.write("\n" + n1 + " * " + n2 + " = " + ( n1 * n2 ) );
    document.write("\n" + n1 + " / " + n2 + " = " + ( n1 / n2 ) );
    document.write("\n" + n1 + " % " + n2 + " = " + ( n1 % n2 ) );
    document.write("\n" + n1 + " ^ " + n2 + " = " + ( parseInt( Math.pow( n1 , n2 ) ) ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 set 21 22:52
